docs/examples/livekit/livekit-server.yaml

```diff
@@ -85,7 +85,7 @@ data:
       port_range_start: 50000
       tcp_port: 7801
       stun_servers:
-        - stunner_ip:3478
+      - stunner_ip:3478
```

Comment:
```suggestion
      - stunner_ip: 3478
```

Reply:
@levaitamas  This is supposed to be a stun server address. The `stunner_ip` string is replaced with a  `sed` command, not a key-value pair.

Reply:
Thanks for pointing that out! I interpreted this line as a yaml key:value. But it is clearly an IP:port 🤪